client/src/components/modals/IncorrectGrantModal.js

```diff
@@ -0,0 +1,71 @@
+import api from 'api'
+import { Box, Button, Heading, Text, TextArea } from 'grommet'
+import * as React from 'react'
+import { Modal } from '../Modal'
+
+export const IncorrectGrantModalContent = ({ setShowing, setAlert }) => {
+  const [message, setMessage] = React.useState('')
+  const onChange = (newMessage) => {
+    setMessage(newMessage)
+  }
+  const onClick = () => {
```

Comment:
could we make this async, await the http response, then if there was an error sending we alert an error saying the issue was not reported and then just continue from there

Reply:
Yep! Done.